processing_handler_test.go

```diff
@@ -721,6 +724,73 @@ func (s *ProcessingHandlerTestSuite) TestModifiedSinceReqCompareTooOldLastModifi
 
 	require.Equal(s.T(), 200, res.StatusCode)
 }
+
+func (s *ProcessingHandlerTestSuite) TestAlwaysRasterizeSvg() {
+	config.AlwaysRasterizeSvg = true
+
+	rw := s.send("/unsafe/rs:fill:40:40/plain/local:///test1.svg")
+	res := rw.Result()
+
+	require.Equal(s.T(), 200, res.StatusCode)
+	require.Equal(s.T(), "image/png", res.Header.Get("Content-Type"))
+
+	actual := s.readBody(res)
+	expected := s.readTestFile("test1.fromsvg.png")
+
+	require.True(s.T(), bytes.Equal(expected, actual))
```

Comment:
It's better to just check `Content-Type` header. Otherwise, this test may fail because of changes in librsvg or libspng. Same for the rest of the tests

Reply:
You are right, I droped the requires and test data that I added